tests/taglib/test_aiff.rs

```diff
@@ -74,6 +77,7 @@ fn test_save_id3v2() {
 		tfile.insert_tag(tag);
 		file.rewind().unwrap();
 		tfile.save_to(&mut file).unwrap();
+		assert!(!tfile.contains_tag_type(lofty::TagType::ID3v2));
```

Comment:
```
running 1 test
thread 'test_aiff::test_save_id3v2' panicked at 'assertion failed: !tfile.contains_tag_type(lofty::TagType::ID3v2)', tests/taglib/test_aiff.rs:80:9
stack backtrace:
   0: rust_begin_unwind
             at /rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/core/src/panicking.rs:142:14
   2: core::panicking::panic
             at /rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/core/src/panicking.rs:48:5
   3: taglib::test_aiff::test_save_id3v2
             at ./tests/taglib/test_aiff.rs:80:3
   4: taglib::test_aiff::test_save_id3v2::{{closure}}
             at ./tests/taglib/test_aiff.rs:49:1
   5: core::ops::function::FnOnce::call_once
             at /rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/core/src/ops/function.rs:248:5
   6: core::ops::function::FnOnce::call_once
             at /rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/core/src/ops/function.rs:248:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
test test_aiff::test_save_id3v2 ... FAILED
```

Reply:
But if I comment out this line [similar test in line 90](https://github.com/Serial-ATA/lofty-rs/pull/51/commits/d251a589a4a987a796d875f6ddc9c995d3d1a8f3#diff-064badcdbd3559333d100cb419d9cc1fe1fd7a9484d3babcc2412d992dbb9fa8R90) runs successfully. Is this made on porpuse or is it a bug @Serial-ATA?

Reply:
Looks like TagLib removes them in `File::save`. Its not purposeful, just didn't think of doing it that way. I'll add it soon 🙂.

Reply:
And something analogus to how TagLib handles files would be very useful too. The File instance would then needed to be stored somewhere inside.

Reply:
Do you mean tying a `TaggedFile` to the `File` it was read from? Something like that could be pretty easily accomplished with a wrapper. I could add that as `BoundTaggedFile` maybe? I don't want that to be the default behavior though, to keeps things reusable.

Reply:
Yes, something like that. I was also thinking about wrapper that would allowed that and provide backwards compatibility.